### ISSUE TITLE
Missed Notification for a starred project

### ISSUE THREAD

**Issue by ainola** [sibbell/sibbell-issues#2]
Hi, thanks for this useful tool.

I have only six projects starred: 

>owncloud / core
csound / csound
kunstmusik / blue
ether / etherpad-lite
residualvm / residualvm
candy-chat / candy

I got a notification a few weeks ago about etherpad-lite getting an update but I missed owncloud 8's release. Looking at 'recent updates' part of Sibbell shows Etherpad and Candy chat as the latest updates. Candy hasn't been updated since July! Meanwhile, Residualvm had an update about a month ago and is not shown as a recent update.

**Comment by davegaeddert** [comment]
Thanks for submitting on issue on this @ainola, and thanks for all the detail! I'll look into it.

**Comment by ComeBurguburu** [comment]
Same bug for
PHPOffice/PHPExcel
The last version show in Sibbel is not the last one

**Comment by maelfrize** [comment]
Same here for https://github.com/angular/material

Sibbell hasn't sent notifications since 0.6.0

**Comment by davegaeddert** [comment]
Hey guys, thanks for reporting this, looks like there's a couple of things going on here. It all has to deal with the loose relationship between "GitHub releases" and git tags.

@maelfrize with angular/material it looks like the last published "GitHub release" was 0.6.0. Since then there have been a number of tags, but they haven't published them as releases (as defined by GitHub). On the GitHub website they do a good job of listing all of the published releases and unpublished git tags next to each other (https://github.com/angular/material/releases) but the APIs aren't setup to work this way so there's quite a bit of extra leg work for me to do to mimic that. Currently Sibbell checks for any releases, and grabs the latest one if there are any, and if there aren't any releases then it grabs the latest tag - so since there's one release (0.6.0) then it doesn't check the tags.

@ComeBurguburu with PHPOffice/PHPExcel there's a different problem. They don't publish any "GitHub releases" so Sibbell looks at the list of tags (https://api.github.com/repos/phpoffice/phpexcel/tags). The API for tags returns them in the same order that git would, by order of name. Since they named one `PHPExcel_1.7.8`, it throws off all of the ordering!
Related StackOverflow question here: http://stackoverflow.com/questions/19452244/github-api-v3-order-tags-by-creation-date

Long story short, it looks like there's some more work to do to try and order all the releases as intelligently as possible, like GitHub does on the website. I'll see what I can do.

**Comment by maelfrize** [comment]
Thank you for clarifying.
And thanks a lot for your efforts.
